obspy/imaging/maps.py

```diff
@@ -272,6 +272,9 @@ def linspace2(val1, val2, N):
     scatter = bmap.scatter(x, y, marker=marker, s=size, c=color,
                            zorder=10, cmap=colormap)
 
+    # Hack!!
```

Comment:
How can this work anyway, I don't see that `ax.basemap` is available normally, i.e. `Axes` are not aware of the `Basemap` instance, only the other way around. And I don't see where it gets attached manually.. or am I missing something..?